challenges/objects-arrays.js

```diff
@@ -80,8 +113,14 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+const newNames = zooAnimals.map((array, animal_name) => {
+  return lowerCase.push(`${array.animal_name.toLowerCase()}`);
+  });
 console.log(lowerCase); 
 
 // The zoos need to know their total animal population across the United States.  Add up all the population numbers from all the zoos using the .reduce() method.
 let populationTotal = [];
+const newName = zooAnimals.map((array, animal_name) => {
```

Comment:
If you come back to this sprint challenge, I'd like to see you tackle this one using .reduce() :smile: